### DISCUSSION TITLE
Support for 7.1 and multichannel streams

### DISCUSSION THREAD

**Discussion by theotherjenkutler** [savonet/liquidsoap#4279] [Ideas]
Hi.
I was wondering if this would even be possible on an icecast2 server.

**Comment by toots** [comment]
It should definitely be possible using the `%ffmpeg` encoder. The radiofrance code has support for it I believe: https://github.com/radiofrance/rf-liquidsoap/blob/master/scripts/transcoder/formats/hls.aac.liq#L57
